### PR TITLE
Drop outdated references to PJAX

### PR DESCRIPTION
**Pull request by fregante**
- Closes https://github.com/refined-github/refined-github/issues/5751#issuecomment-1275907968

The `#js-repo-pjax-container` continues to exist though

Untested

### CODE REVIEW

File: source/features/default-branch-button.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ async function init(): Promise<false | void> {
 		<a
 			className="btn tooltipped tooltipped-ne rgh-default-branch-button"
 			href={url.href}
-			data-pjax="#repo-content-pjax-container"
+			data-turbo-frame="repo-content-turbo-frame"
```
Comment (fregante) [inline source/features/default-branch-button.tsx:40]: Seen on some native links:

Screen Shot

**Comment by yakov116** [comment]
I am getting a whole bunch of error after this PR

For example refined-github.js:3726 ❌ mark-merge-commits-in-list

**Comment by fregante** [comment]
The PR doesn't even touch that feature

**Comment by yakov116** [comment]
Sorry looks like GitHub is in middle of changing the commits to be lazy loading.
I wanted to test if it was this that caused the issue so I built the project prior to the commit and it was working fine. But now I see it all over.

Sorry for the confusion

**Pull request merged**